cmd/monitorcodegen/main.go

```diff
@@ -58,7 +58,11 @@ const (
 
 var metricSet = map[string]monitors.MetricInfo {
 {{- range .metrics}}
-	{{.Name | formatVariable}}: {Type: {{.Type | convertMetricType}}},
+	{{- if .Group}}
+		{{.Name | formatVariable}}: {Type: {{.Type | convertMetricType}}, Group: {{.Group | deref | printf "group.%s" | formatVariable}}},
```

Comment:
Had to defef `.Group` here, but couldn't really figure out why `.Group` is shown as a pointer in printf statement.